model_compression_toolkit/target_platform_capabilities/schema/v1.py

```diff
@@ -697,4 +697,5 @@ def show(self):
         """
         Display the TargetPlatformCapabilities.
         """
-        pprint.pprint(self.get_info(), sort_dicts=False)
\ No newline at end of file
+        pprint.pprint(self.get_info(), sort_dicts=False)
+
```

Comment:
remove new line (just because it is a shema file, it is better to not have unnecessary "messy" history, otherwise I wouldn't care)